components/elm/src/main/elmfates_interfaceMod.F90

```diff
@@ -515,23 +530,56 @@ subroutine ELMFatesGlobals2()
            pass_lu_harvest = 0
            pass_num_lu_harvest_types = 0
         end if
-        call set_fates_ctrlparms('use_lu_harvest',ival=pass_lu_harvest)
-        call set_fates_ctrlparms('num_lu_harvest_cats',ival=pass_num_lu_harvest_types)
-        call set_fates_ctrlparms('use_logging',ival=pass_logging)
 
         if(use_fates_luh) then
            pass_use_luh = 1
            pass_num_luh_states = num_landuse_state_vars
            pass_num_luh_transitions = num_landuse_transition_vars
+
+           ! Do not set harvest passing variables to zero not in luh harvest modes
+           ! as the user may want to use the CLM landuse harvest with luh2 transitions
+           if(fates_harvest_mode >= fates_harvest_luh_area) then
+              ! End the run if do_harvest is true with this run mode.
+              ! This should be caught be the build namelist.
+              if(get_do_harvest()) then
+                 call endrun(msg="do_harvest and fates_harvest_mode using luh2 harvest data are incompatible"//&
+                      errmsg(sourcefile, __LINE__))
+              else
+                 pass_num_lu_harvest_types = num_landuse_harvest_vars
+                 pass_lu_harvest = 1
+              end if
+           end if
+
         else
            pass_use_luh = 0
            pass_num_luh_states = 0
            pass_num_luh_transitions = 0
         end if
+
         call set_fates_ctrlparms('use_luh2',ival=pass_use_luh)
         call set_fates_ctrlparms('num_luh2_states',ival=pass_num_luh_states)
         call set_fates_ctrlparms('num_luh2_transitions',ival=pass_num_luh_transitions)
 
+        if ( use_fates_potentialveg ) then
+           pass_use_potentialveg = 1
+        else
+           pass_use_potentialveg = 0
+        end if
+        call set_fates_ctrlparms('use_fates_potentialveg',ival=pass_use_potentialveg)
+
+        if(flandusepftdat /= '') then
+           pass_lupftdat = 1
+        else
+           pass_lupftdat = 0
+        end if
+        call set_fates_ctrlparms('use_landusepft_data',ival=pass_lupftdat)
```

Comment:
Is `use_landusepft_data ` unused?

Reply:
Currently it is unused in fates.  I need to check this against the fates usage.  We might be able to remove it.

Reply:
Conferring with @ckoven, this will be removed.

Reply:
(Resolved)